templates/single-product/related.php

```diff
@@ -46,9 +46,9 @@
 
 if ( $products->have_posts() ) : ?>
 
-	<div class="related products">
+	<section class="related products">
 
-		<h2><?php _e( 'Related Products', 'woocommerce' ); ?></h2>
+		<h2><?php echo sprintf( __( 'Other %s...', 'woocommerce' ), $product->get_categories() ); ?></h2>
```

Comment:
What default category?
We can get the first, still I think that a lot of people will be sad with the wrong category :P